packages/firestore/test/unit/local/local_store_indexeddb.test.ts

```diff
@@ -423,7 +436,7 @@ describe('LocalStore w/ IndexedDB Persistence (Non generic)', () => {
 
     await test.executeQuery(queryCount);
     test.assertRemoteDocumentsRead(2, 0);
-    test.assertOverlaysRead(2, 0);
+    test.assertOverlaysRead(2, 0, {});
```

Comment:
what's the difference between this case and the above test? (why isn't this one having MutationType.Delete?)